lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -359,17 +359,19 @@ sub job_next_previous_ajax {
         my $bugs       = $labels->{$id}{bugs};
         my $bugdetails = $labels->{$id}{bugdetails};
 
-        my (@bugs, @bug_urls, @bug_icons);
+        my (@bugs, @bug_urls, @bug_icons, @bug_titles);
         for my $bug (sort { $b cmp $a } keys %$bugs) {
-            push @bugs,      $bug;
-            push @bug_urls,  $self->bugurl_for($bug);
-            push @bug_icons, $self->bugicon_for($bug, $bugdetails->{$bug});
+            push @bugs,       $bug;
+            push @bug_urls,   $self->bugurl_for($bug);
+            push @bug_icons,  $self->bugicon_for($bug, $bugdetails->{$bug});
+            push @bug_titles, $self->bugtitle_for($bug, $bugdetails->{$bug});
```

Comment:
Yet another array. This should be one array containing of hashrefs.

Reply:
updated patch